src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,36 @@
 
 class Animal {
   // write your code here
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
```

Comment:
this method should only hide
```suggestion
  hide() {
    this.hidden = true;
  }
```